src/Controller/Front/UserController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Controller\Front;
+
+use App\Entity\User;
+use App\Form\UserType;
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+use App\Entity\Animal;
+use App\Form\AnimalType; 
+use App\Repository\AnimalRepository;
+
+/**
+ * @Route("/user", name="user_")
+ */
+class UserController extends AbstractController
+{
+
+    /**
+     * @Route("/{id}", name="user_show", methods={"GET"})
+     */
+    public function show(User $user): Response
+    {
+        return $this->render('user/show.html.twig', ['user' => $user]);
+    }
+
+    /**
+     * @Route("/{id}/edit", name="edit", methods={"GET","POST"})
+     */
+    public function edit(Request $request, User $user): Response
+    {
+        $form = $this->createForm(UserType::class, $user);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->getDoctrine()->getManager()->flush();
+
+            return $this->redirectToRoute('user_index', ['id' => $user->getId()]);
+        }
+
+        return $this->render('user/edit.html.twig', [
+            'user' => $user,
+            'form' => $form->createView(),
+        ]);
+    }
+
+    /**
+     * @Route("/{id}/animals", name="animals", methods={"GET"})
+     */
+    public function show_animals(User $user, AnimalRepository $animalRepository): Response
```

Comment:
https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions
```suggestion
    public function showAnimals(User $user, AnimalRepository $animalRepository): Response
```